src/lib/Syntax.hs

```diff
@@ -843,9 +843,8 @@ type UConDef (n::S) (l::S) = (SourceName, Nest (UAnnBinder AtomNameC) n l)
 -- than being scoped names of the proper color of their own?
 data UDataDef (n::S) where
```

Comment:
So we have interface constraints in the surface language but not the core IR, is that right?